html/changelogs/Spud5-physical_status_missing.yml

```
@@ -0,0 +1,9 @@
+
+author: Spud2
+
+
+delete-after: True
+
+
+changes:
+  - rscadd: "Adds "*Missing*" as a possible physical status in medical records."
```

```suggestion
  - rscadd: "Adds \"*Missing*\" as a possible physical status in medical records."
```
escape characters necessary.